package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "local-home-virual-environment",
+  "name": "local-home-virtual-environment",
   "version": "0.0.1",
   "description": "",
   "private": true,
```

Comment:
remove private?

Reply:
I think we should keep as private until we're ready to distribute the artifact.

Reply:
Will leave as private for now